<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpen">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHelp">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Справка</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
